### ISSUE TITLE
The config attributes {'shift_factor': None, 'use_post_quant_conv': True, 'use_quant_conv': True} were passed to AutoencoderKL, but are not expected and will be ignored. Please verify your config.json configuration file.

### ISSUE THREAD

**Issue by bule557** [lllyasviel/Paints-UNDO#101]
"Steps 1, 2, and 3 all run normally and generate images and videos, but there is this part with a 'The config attributes {'shift_factor': None, 'use_post_quant_conv': True, 'use_quant_conv': True} were passed to AutoencoderKL, but are not expected and will be ignored. Please verify your config.json configuration file..', does it affect the generated content, and can it be commented out?"

``The config attributes {'shift_factor': None, 'use_post_quant_conv': True, 'use_quant_conv': True} were passed to AutoencoderKL, but are not expected and will be ignored. Please verify your config.json configuration file.
Loading weights from local directory
Loading weights from local directory
Loading weights from local directory
Unload to CPU: ImprovedCLIPVisionModelWithProjection
Unload to CPU: AutoencoderKL
Unload to CPU: UNet3DModel
Unload to CPU: VideoAutoencoderKL
Unload to CPU: Resampler
Unload to CPU: CLIPTextModel
Unload to CPU: CLIPTextModel
Unload to CPU: ModifiedUNet
/data/coding/Paints-UNDO/diffusers_helper/k_diffusion.py:43: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  alphas_cumprod = torch.tensor(np.cumprod(alphas, axis=0), dtype=torch.float32)
Running on local URL:  http://0.0.0.0:7860

To create a public link, set `share=True` in `launch()`.
/data/miniconda/envs/paints_undo/lib/python3.10/site-packages/gradio/analytics.py:106: UserWarning: IMPORTANT: You are using gradio version 4.43.0, however version 4.44.1 is available, please upgrade.
